### PR TITLE
trying to fix ci

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *The dns on the CI is broken*

## Solution
- *Try a workaround (if someone can fix the CI it would be better)*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/baikal_ynh%20PR45/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/change_url [hunk 1]
```diff
@@ -52,7 +52,7 @@ fi
 if [ $change_domain -eq 1 ]
 then
 	# Check if .well-known is available for the new domain.
-	if is_url_handled "https://$new_domain/.well-known/caldav" || is_url_handled "https://$new_domain/.well-known/carddav"
+	if is_url_handled "$new_domain" "/.well-known/caldav" || is_url_handled "$new_domain" "/.well-known/carddav"
```
Comment (maniackcrudelis) [inline scripts/change_url]: Let's use getopts arguments

**Comment by maniackcrudelis** [comment]
Looks ok
If it works (looks ok, but let's wait until the end of the test), could you update the helper as well ?
https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/is_url_handled/is_url_handled__2

**Comment by kay0u** [comment]
> Looks ok
> If it works (looks ok, but let's wait until the end of the test), could you update the helper as well ?
> https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/is_url_handled/is_url_handled__2

Done

**Comment by kay0u** [comment]
Bump

**Comment by kay0u** [comment]
Yolomergin in 3 days if nobody is against it

**Pull request merged**